From 9cdb08a0d6fbce2b8621b3e3296045512d26e7c2 Mon Sep 17 00:00:00 2001
From: Thomas Goirand <zigo@debian.org>
Date: Tue, 15 Sep 2015 18:30:11 -0400
Subject: Do not use the bootstrap sphinx theme

 This patch removes the use of the bootstrap sphinx theme which isn't yet in
 Debian (and for which I have no time to work on).
Forwarded: not-needed
Last-Update: 2015-04-10

Patch-Name: do-not-use-bootstrap.patch
---
 docs/_templates/sidebartoc.html |  2 +-
 docs/conf.py                    | 74 -----------------------------------------
 2 files changed, 1 insertion(+), 75 deletions(-)

diff --git a/docs/_templates/sidebartoc.html b/docs/_templates/sidebartoc.html
index 0d119af..2edf636 100644
--- a/docs/_templates/sidebartoc.html
+++ b/docs/_templates/sidebartoc.html
@@ -1 +1 @@
-{{ toctree(maxdepth=theme_globaltoc_depth|toint, collapse=True, includehidden=theme_globaltoc_includehidden|tobool) }}
+{{ toctree(maxdepth=toint, collapse=True, includehidden=theme_globaltoc_includehidden|tobool) }}
diff --git a/docs/conf.py b/docs/conf.py
index 67e92a8..df36abe 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -14,7 +14,6 @@
 from __future__ import absolute_import, print_function
 import sys, os
 from future import __version__
-import sphinx_bootstrap_theme
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
@@ -98,79 +97,6 @@ pygments_style = 'sphinx'  # 'futureext.FutureStyle'
 
 
 # -- Options for HTML output ---------------------------------------------------
-
-# The theme to use for HTML and HTML Help pages.  See the documentation for
-# a list of builtin themes.
-html_theme = 'bootstrap'
-html_theme_path = sphinx_bootstrap_theme.get_html_theme_path()
-
-# Theme options are theme-specific and customize the look and feel of a theme
-# further.  For a list of options available for each theme, see the
-# documentation.
-html_theme_options = {
-    # Navigation bar title. (Default: ``project`` value)
-    #'navbar_title': "Python-Future",
-
-    # Tab name for entire site. (Default: "Site")
-    'navbar_site_name': "Contents",
-
-    # A list of tuples containing pages or urls to link to.
-    # Valid tuples should be in the following forms:
-    #    (name, page)                 # a link to a page
-    #    (name, "/aa/bb", 1)          # a link to an arbitrary relative url
-    #    (name, "http://example.com", True) # arbitrary absolute url
-    # Note the "1" or "True" value above as the third argument to indicate
-    # an arbitrary url.
-    'navbar_links': [
-        ("Overview", "overview"),
-        ("Cheat Sheet", "compatible_idioms.html", True),
-        ("FAQ", "faq.html", True),
-        # ("Link", "http://example.com", True),
-    ],
-
-    # Render the next and previous page links in navbar. (Default: true)
-    'navbar_sidebarrel': False,
-
-    # Render the current pages TOC in the navbar. (Default: true)
-    'navbar_pagenav': True,
-
-    # Global TOC depth for "site" navbar tab. (Default: 1)
-    # Switching to -1 shows all levels.
-    'globaltoc_depth': 3,
-
-    # Include hidden TOCs in Site navbar?
-    #
-    # Note: If this is "false", you cannot have mixed ``:hidden:`` and
-    # non-hidden ``toctree`` directives in the same page, or else the build
-    # will break.
-    #
-    # Values: "true" (default) or "false"
-    'globaltoc_includehidden': "true",
-
-    # HTML navbar class (Default: "navbar") to attach to <div> element.
-    # For black navbar, do "navbar navbar-inverse"
-    'navbar_class': "navbar navbar-inverse",
-
-    # Fix navigation bar to top of page?
-    # Values: "true" (default) or "false"
-    'navbar_fixed_top': "true",
-
-    # Location of link to source.
-    # Options are "nav" (default), "footer" or anything else to exclude.
-    'source_link_position': "none",
-
-    # Bootswatch (http://bootswatch.com/) theme.
-    #
-    # Options are nothing with "" (default) or the name of a valid theme
-    # such as "amelia" or "cosmo" or "united".
-    'bootswatch_theme': "cerulean",
-
-    # Choose Bootstrap version.
-    # Values: "3" (default) or "2" (in quotes)
-    'bootstrap_version': "3",
-}
-
-
 # Add any paths that contain custom themes here, relative to this directory.
 #html_theme_path = []
 
